### ISSUE TITLE
[Accessibility] Focus indicator is missing

### ISSUE THREAD

**Issue by michalina-graczyk** [mirumee/saleor-storefront#834]
### What I'm trying to achieve is to have The visual focus indicator.

Summary | Selector
-- | --
The visual focus indicator is missing altogether. | ["body > div#root:nth-of-type(1) > div.home-page:nth-of-type(1) > div.home-page__hero:nth-of-type(1) > div.home-page__hero-action:nth-of-type(2) > a:nth-of-type(1)"]
The visual focus indicator is missing altogether. | ["body > div#root:nth-of-type(1) > div.home-page:nth-of-type(1) > div.home-page__hero:nth-of-type(1) > div.home-page__hero-action:nth-of-type(2) > a:nth-of-type(1) > button.button:nth-of-type(1)"]

**System information**
Operating system: MacOs
Browser: Chrome v.84
